README.md

```diff
@@ -255,14 +234,22 @@ Perform following commands
 
 # NodeJS
 
-## Practical 15
+## Practical 13
 
 - Real-time group chat application
-  
-  - Build a real-time group chat application using Node.js, Javascript, and a library like Socket.IO.
+  - Build a real-time group chat application using Node.js, TypeScript, and a library like Socket.IO.
   - Allow users to join chat rooms, send messages, and receive messages in real time.
 
-## Practical 16
+## Practical 14
+- Quiz Application with Firebase Realtime Database
+  - This web application allows users to answer the quizzes and tracks their scores in real-time.
+  - Store multiple-choice questions and their correct answers in the database using Typescript. Create quizzes based on categories(Like science, entertainment etc...).
```

Comment:
Let's simplify it by
```
The application will have two sides.
1. Create Quiz
2. Play Quiz

- Create Quiz
  - Authenticate user with firebase authentication and allow to create a quiz with MCQ along with correct option
- Play Quiz
  - Authenticate user and firebase authentication and allow to play quiz along with storing results(scores)
  - Display score at the end of the Quiz 
```

Reply:
I think we should add categories, it's not that complex at this level.

For  `Display score at the end of the Quiz `
I guess answering each question will be fine, as we want to show real-time changes